pkg/workloadmgrs/k8swm/k8swm.go

```diff
@@ -89,6 +87,8 @@ func (m *K8sWorkloadMgr) CreateAppInst(ctx context.Context, clusterInst *edgepro
 
 			err = k8smgmt.WaitForAppInst(ctx, client, names, app, k8smgmt.WaitRunning)
 		}
+	case cloudcommon.DeploymentTypeHelm:
```

Comment:
How are we dealing with namespaces in helm? Do we need a TODO here to properly handle this? From what I understood we'll need vcluster to properly handle multi-tenant cluster with namespaces